src/Dropdown/Dropdown.tsx

```diff
@@ -4,25 +4,41 @@
  * SPDX-License-Identifier: LicenseRef-Nordic-4-Clause
  */
 
-import React, { useEffect, useState } from 'react';
+import React, { useEffect, useRef, useState } from 'react';
 import FormLabel from 'react-bootstrap/FormLabel';
 import { arrayOf, bool, func, number, shape, string } from 'prop-types';
 
 import chevron from './chevron.svg';
 
 import styles from './Dropdown.module.scss';
 
-interface DropdownItem {
+const useSynchronisationIfChangedFromOutside: <T>(
+    v: T,
+    set: (value: T) => void
+) => T = (externalValue, setInternalValue) => {
+    const previousExternalValue = useRef(externalValue);
+    useEffect(() => {
+        if (previousExternalValue.current !== externalValue) {
+            setInternalValue(externalValue);
+            previousExternalValue.current = externalValue;
+        }
+    });
+    return previousExternalValue.current;
+};
+
+export interface DropdownItem {
     label: string;
     value: string;
 }
 
-interface DropdownProps {
+export interface DropdownProps {
     label?: string;
     items: DropdownItem[];
     onSelect: (item: DropdownItem) => void;
     disabled?: boolean;
     defaultIndex?: number;
+    selectedItem?: DropdownItem;
+    noItemsBeforeScroll?: number;
```

Comment:
```suggestion
    ItemCountBeforeScroll?: number;
```
I kept reading it as "no items before scroll" even after realising it was "number ..."
Not sure if my naming suggestion is anymore intuitive though...

Reply:
how about num to keep it short?

Reply:
Not sure if you are genius or if I'm blind. I think that's perfect